projects/aca-shared/src/lib/components/toolbar/toolbar.component.scss

```diff
@@ -0,0 +1,27 @@
+aca-toolbar {
+  .mat-toolbar {
+    min-height: 48px;
+    color: var(--adf-theme-foreground-text-color-054);
+    background-color: inherit;
+    border: none;
+    padding: 0;
+  }
+
+  .mat-toolbar-row {
+    height: 48px;
+    font-size: var(--theme-body-1-font-size);
+    white-space: normal;
+  }
+
+  .mat-toolbar-single-row {
+    height: 64px;
+  }
+
+  .aca-toolbar-divider {
+    height: 24px;
+    width: 1px;
+    background: rgba(0, 0, 0, 0.26);
```

Comment:
we probably should use some already defined colors from variables?

Reply:
good point, copied from the ADF, will fix